src/library/Uno.Themes.WinUI.Markup/Theme/Typography.cs

```diff
@@ -0,0 +1,221 @@
+﻿using System;
+using Microsoft.UI.Xaml.Media;
+using Uno.Extensions.Markup.Internals;
+using Windows.UI.Text;
+
+namespace Uno.Themes.Markup
+{
+	public static partial class Theme
+	{
+		public static class Typography
+		{
+			public static class Resources
```

Comment:
This is a special case, the Resources class doesn't need to be here. It should be `Theme.Typography.DisplayLarge.FontWeight/FontSize/etc.`